pom.xml

```diff
@@ -78,12 +78,6 @@
 			<groupId>eu.europeana.metis</groupId>
 			<artifactId>metis-common</artifactId>
 			<version>${version.metis}</version>
-			<exclusions>
-				<exclusion>
-					<groupId>org.apache.logging.log4j</groupId>
-					<artifactId>log4j-slf4j-impl</artifactId>
-				</exclusion>
-			</exclusions>
 		</dependency>
 
 		<dependency>
```

Comment:
How about this dependency? Should the exclusion not be removed here either?

Reply:
That is what I thought, and I did try, but that service depends on `corelib-edm-definitions`, which [parent](https://github.com/europeana/corelib/blob/develop/pom.xml) has the dependency

Reply:
I see, thanks. It should therefore be excluded there, not here. I'll change it in metis.